### PR TITLE
Hk develop #3

### PR DESCRIPTION
**Pull request by HerbertIV**

### CODE REVIEW

File: src/EscolaLmsCategoriesServiceProvider.php [hunk 1]
```diff
@@ -27,5 +28,7 @@ public function boot()
     {
         $this->loadRoutesFrom(__DIR__ . '/routes.php');
         $this->loadMigrationsFrom(__DIR__ . '/../database/migrations');
+        $router = $this->app['router'];
+        $router->aliasMiddleware('role', \Spatie\Permission\Middlewares\RoleMiddleware::class);
```
Comment (MWL91) [inline src/EscolaLmsCategoriesServiceProvider.php:32]: `$this->app['router']->aliasMiddleware('role', \Spatie\Permission\Middlewares\RoleMiddleware::class);`

File: tests/API/CategoriesApiTest.php [hunk 2]
```diff
@@ -58,13 +63,55 @@ public function testCategoryCreate(): void
...
     }
+
+    public function testCategoryUpdateUserStudent(): void
+    {
+        $user = config('auth.providers.users.model')::factory()->create();
+        $user->assignRole('student');
+        Passport::actingAs($user, ['categories']);
+
+        $category = Category::factory()->create();
+        $this->response = $this->startSession()->json('PUT', '/api/categories/' . $category->getKey(), [
```
Comment (MWL91) [inline tests/API/CategoriesApiTest.php:85]: Można by dać po prostu `$this->actingAs($user, 'api')->json`

**Review by MWL91 — APPROVED** [review]

File: tests/API/CategoriesApiTest.php (hunk 2 above)
Comment (HerbertIV) [reply tests/API/CategoriesApiTest.php]: Zmienione

File: src/EscolaLmsCategoriesServiceProvider.php (hunk 1 above)
Comment (HerbertIV) [reply src/EscolaLmsCategoriesServiceProvider.php]: Zmienione

**Pull request merged**